WcaOnRails/app/views/results/_results_selector.html.erb

```diff
@@ -86,15 +86,16 @@
   </div>
 </div>
 
-<% if show_rankings_options %>
-  <div class="form-group">
-    <%= label_tag(t("results.selector_elements.gender_selector.gender")) %>
-    <div id="gender" class="btn-group">
-      <%= link_to t("results.selector_elements.gender_selector.gender_all"), request.params.merge(gender: "All"), class: "btn btn-info" + (@gender == "All" ? " active" : "") %>
-      <%= link_to t("results.selector_elements.gender_selector.male"), request.params.merge(gender: "Male"), class: "btn btn-info" + (@gender == "Male" ? " active" : "") %>
-      <%= link_to t("results.selector_elements.gender_selector.female"), request.params.merge(gender: "Female"), class: "btn btn-info" + (@gender == "Female" ? " active" : "") %>
-    </div>
+<div class="form-group">
+  <%= label_tag(t("results.selector_elements.gender_selector.gender")) %>
+  <div id="gender" class="btn-group">
+    <%= link_to t("results.selector_elements.gender_selector.gender_all"), request.params.merge(gender: "All"), class: "btn btn-info" + (@gender == "All" ? " active" : "") %>
+    <%= link_to t("results.selector_elements.gender_selector.male"), request.params.merge(gender: "Male"), class: "btn btn-info" + (@gender == "Male" ? " active" : "") %>
+    <%= link_to t("results.selector_elements.gender_selector.female"), request.params.merge(gender: "Female"), class: "btn btn-info" + (@gender == "Female" ? " active" : "") %>
   </div>
+</div>
+
+<% if show_rankings_options %>
```

Comment:
This change looks odd in the GitHub diff. What effectively happens is that I moved the gender selectbox out of the `if show_rankings_options` block, so that it is always displayed by default. The entire inner block with nested divs is unchanged.

Reply:
FTR: it looks great if you check the "hide whitespace change" option ;)